x-pack/plugins/apm/server/feature.ts

```diff
@@ -72,6 +73,60 @@ export const APM_FEATURE = {
       ui: ['show', 'alerting:show', 'alerting:save'],
     },
   },
+  subFeatures: [
+    {
+      name: i18n.translate('xpack.apm.featureRegistry.manageAlertsName', {
+        defaultMessage: 'Alerts',
+      }),
+      privilegeGroups: [
+        {
+          groupType: 'mutually_exclusive' as SubFeaturePrivilegeGroupType,
+          privileges: [
+            {
+              id: 'alerts_all',
+              name: i18n.translate(
+                'xpack.apm.featureRegistry.subfeature.alertsAllName',
+                {
+                  defaultMessage: 'All',
+                }
+              ),
+              includeIn: 'all' as 'all',
+              alerting: {
+                alert: {
+                  all: Object.values(AlertType),
+                },
+              },
+              savedObject: {
+                all: [],
+                read: [],
+              },
+              ui: [],
+            },
+            {
+              id: 'alerts_read',
+              name: i18n.translate(
+                'xpack.apm.featureRegistry.subfeature.alertsReadName',
+                {
+                  defaultMessage: 'Read',
+                }
+              ),
+              includeIn: 'read' as 'read',
+              alerting: {
+                alert: {
+                  read: Object.values(AlertType),
```

Comment:
This alerting read privilege should be removed from the top-level `all` and `read` privileges above, since this is being granted by a sub-feature privilege. Positive and negative end-to-end tests asserting this behavior should also be added if they don't already exist.

Reply:
Thanks for this spot!

Fixed here 6111991397dd51f51afab7c6a2cda30db267b357

Additional tests added here c38a0031670f5866b5ffc1300696dcc2f54f7b16